sdk/healthdataaiservices/Azure.ResourceManager.HealthDataAIServices/api/Azure.ResourceManager.HealthDataAIServices.netstandard2.0.cs

```diff
@@ -257,15 +258,28 @@ public partial class HealthDataAIServicesPrivateLinkServiceConnectionState : Sys
     }
+    public partial class ManagedServiceIdentity : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentity>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentity>
+    {
+        public ManagedServiceIdentity(Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentityType type) { }
+        public System.Guid? PrincipalId { get { throw null; } }
+        public System.Guid? TenantId { get { throw null; } }
+        public Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentityType Type { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, Azure.ResourceManager.Models.UserAssignedIdentity> UserAssignedIdentities { get { throw null; } }
+        Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentity System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentity>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentity>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentity System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentity>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentity>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentity>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public readonly partial struct ManagedServiceIdentityType : System.IEquatable<Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentityType>
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
         public ManagedServiceIdentityType(string value) { throw null; }
         public static Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentityType None { get { throw null; } }
+        public static Azure.ResourceManager.HealthDataAIServices.Models.ManagedServiceIdentityType SystemAndUserAssigned { get { throw null; } }
```

Comment:
need investigation to understand why does this diff come, common-type version issue?

Reply:
Yes this is because of the version of common-types